mycroft/skills/media/vocab/en-us/FromKeyword.voc

```diff
@@ -0,0 +1 @@
+from
```

Comment:
Yeah, and you can even register some regex if you don't want to go through the process of finding every single possible playlist or song name. Something like `from (?P<MediaName>.*)` would match any request to play from somewhere.